config.example.yml

```diff
@@ -21,3 +21,22 @@
     LOG_LEVEL: INFO
     LOG_FILE: bot.log
     LOG_TO_CONSOLE: True
+  TELEGRAM:
+    # set to True to enable telegram notifications
+    ENABLED: False
+    # Getting a bot token: https://core.telegram.org/bots#6-botfather
+    BOT_TOKEN: my_token
+    # Getting your chatID: find your bot and send a message,
+    # then go visit https://api.telegram.org/bot<YOUR_TOKEN>/getUpdates and find the chat ID
+    BOT_CHAT_ID: my_chat_id
+    # Disable / Enable specific notifications
+    NOTIFICATIONS:
+      STARTUP: True # welcome message
+      COIN_ANNOUNCEMENT: True # detected new announcement
+      COIN_NOT_SUPPORTED: True # coin is not on gate.io
+      BUY_START: True # when entering position
+      BUY_ORDER_CREATED: True # when buy order is created
+      BUY_FILLED: True # when the buy order got filled
+      SELL_START: True # when starting to sell      
+      SELL_FILLED: True # when sold
+      DEBUG: True # catch-all for debug messages
```

Comment:
DEBUG is not really used so far since all your telegram statements are named. I guess we can remove this one and keep naming them?

Reply:
I put it there in case someone in the future calls it without a name. Since debug is default paramter it would send as notification. Gonna take it out.